core/src/main/java/nu/yona/server/analysis/service/ActivityService.java

```diff
@@ -209,9 +207,12 @@ private long getTotalPageableItems(Set<BuddyDto> buddies, ChronoUnit timeUnit)
 	private long getTotalPageableItems(UserAnonymizedDto userAnonymized, ChronoUnit timeUnit)
 	{
 		long activityMemoryDays = yonaProperties.getAnalysisService().getActivityMemory().toDays();
-		Optional<LocalDateTime> oldestGoalCreationTime = userAnonymized.getOldestGoalCreationTime();
-		long activityRecordedDays = oldestGoalCreationTime.isPresent()
-				? (Duration.between(oldestGoalCreationTime.get(), TimeUtil.utcNow()).toDays() + 1) : 0;
+		Optional<LocalDate> oldestGoalCreationDate = userAnonymized.getOldestGoalCreationTime()
```

Comment:
We could use the suffix `Utc` for variables/fields/properties/methods containing or returning UTC `LocalDateTime`s. That way it is clearer which dates are in UTC and should be converted to user time zone.

Reply:
I believe ``LocalDateTime`` is always UTC in our code, so adding that to the name doesn't add value IMHO.